### PR TITLE
docs: tweaks in README.rst

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR adds some grammar/composition changes to the repo's readme. Please feel free to edit any part of the suggestions before merging them.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,44 +28,44 @@ Plugins List
     .. image:: https://raw.githubusercontent.com/eduNEXT/frontend-plugins-communications/main/screenshots/individual_learners_plugin.png
         :alt: communications-app-individual-emails Image
 
-**communications-app-instructions-pro-freading**
-   :NPM Package: `communications-app-instructions-pro-freading <https://www.npmjs.com/package/@edunext/plugins-communications-app-instructions-pro-freading>`_
+**communications-app-instructions-proof-reading**
+   :NPM Package: `communications-app-instructions-proof-reading <https://www.npmjs.com/package/@edunext/plugins-communications-app-instructions-proof-reading>`_
```
Comment (johnvente) [inline README.rst]: you shouldn't change the package, that one doesn't exits
https://www.npmjs.com/package/@edunext/plugins-communications-app-instructions-proof-reading

Comment (mariajgrimaldi) [reply README.rst]: Yes, but pro-freading isn't a typo? what does pro-freading mean?

Comment (johnvente) [reply README.rst]: It was a mistake, but please don't remove the package

Comment (mariajgrimaldi) [reply README.rst]: Alright. I'll remove the change so it's updated later

**Comment by mariajgrimaldi** [comment]
@johnvente comments have been addressed. Could you check again? Thanks

**Review by johnvente — APPROVED** [review]
LGTM

**Pull request merged**